### ISSUE TITLE
Add a server to an organization

### ISSUE THREAD

**Issue by drew-russell** [rubrikinc/rubrik-sdk-for-python#234]
x-post from https://github.com/rubrikinc/rubrik-modules-for-ansible/issues/94

**Describe the solution you'd like**

I would like to be able to using ansible to add a server to an organization including "Availability Group"

Currently I log into the GUI and upgrade the organizations

**Describe alternatives you've considered**

n/a
**Additional context**

n/a

**Comment by drew-russell** [comment]
`POST /internal/role/354083ab-1310-456a-b9a8-fe1020ce8f59/authorization`

Request Body:
```json
{
  "authorizationSpecifications": [
    {
      "privilege": "ManageRestoreSource",
      "resources": [
        "Host:::8d04f0bb-597d-45cc-ae85-73e0a2b299b5",
        "MssqlAvailabilityGroup:::62be9975-a529-4b9b-bc23-a48238c8ff0c"
      ]
    }
  ],
  "roleTemplate": "Organization"
}
```

Response Body:

```json
{
  "authorizationSpecifications": [
    {
      "privilege": "ManageAccess",
      "resources": [
        "User:::d9d661a0-fbd2-4cda-bf57-6cf14947fa7f"
      ]
    },
    {
      "privilege": "ManageRestoreSource",
      "resources": [
        "Host:::8d04f0bb-597d-45cc-ae85-73e0a2b299b5",
        "MssqlAvailabilityGroup:::62be9975-a529-4b9b-bc23-a48238c8ff0c"
      ]
    }
  ],
  "roleId": "354083ab-1310-456a-b9a8-fe1020ce8f59",
  "roleTemplate": "Organization",
  "organizationId": "97350148-38a6-4ffc-a8a4-5c0a38098a66"
}
```

**Comment by shamsway** [comment]
Resolved by #241 - closing issue
